playbooks/luna_demo_environment.yml

```diff
@@ -0,0 +1,81 @@
+# Manually verify your hosts has FQDN set correctly
+# check it's in /etc/hosts /etc/hostname and hostname returns it, /etc/hosts first line should be 192.168.73.1  $fqdn
+
+# TODO - add role for setting up the machine - autologin in gdm, perhaps background image etc, installation of virt-manager, vim, htop
+# TODO - refactor katello_provisioning so that it does not run installer, it should only seed stuff, installer should be customized from playbook using foreman_installer's role variables
+
+- hosts: all
+  become: true
+  roles:
+    - disable_ipv6
+    - disable_firewall
+    - selinux
+    - etc_hosts
+    - epel_repositories
+    - ostree_repositories
+    - puppet_repositories
+    - foreman_repositories
+    - katello_repositories
+    - foreman_installer
```

Comment:
Need to test, but the foreman installer might be running twice since it's also in foreman_provisioning which is being run by katello_provisioning

Reply:
Agreed. It may be by design, but it certainly slows the playbook.

Reply:
I think the various plugins also rerun the installer which will slow it down. You can probably add more parameters to the main installer and only include the seed parts or the plugins I think.